runtime/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "parity-ethereum-runtime"
+version = "0.1.1"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+description = "Tokio runtime wrapper"
+license = "GPL-3.0"
+readme = "README.md"
+homepage = "https://www.parity.io/"
+keywords = ["parity", "runtime", "tokio"]
+categories = ["embedded"]
```

Comment:
I think `embedded` here refers to [embedded systems](https://en.wikipedia.org/wiki/Embedded_system).
 ```suggestion
```

Reply:
Sorry. I didn't have good ideas, what to insert in categories here, looked into parity common crates, but didn't find good examples

Reply:
categories are used to optimize search on crates.io, maybe we should mentioned this crate is used for legacy reasons if should be avoided by new crates is someone will encounter it